packages/client/components/JiraServerScopingSearchResultsRoot.tsx

```diff
@@ -0,0 +1,44 @@
+import graphql from 'babel-plugin-relay/macro'
+import React, {Suspense} from 'react'
+import {useFragment} from 'react-relay'
+import useQueryLoaderNow from '../hooks/useQueryLoaderNow'
+import jiraServerScopingSearchResultsQuery, {
+  JiraServerScopingSearchResultsQuery
+} from '../__generated__/JiraServerScopingSearchResultsQuery.graphql'
+import {JiraServerScopingSearchResultsRoot_meeting$key} from '../__generated__/JiraServerScopingSearchResultsRoot_meeting.graphql'
+import JiraServerScopingSearchResults from './JiraServerScopingSearchResults'
+
+interface Props {
+  meetingRef: JiraServerScopingSearchResultsRoot_meeting$key
+}
+
+const JiraServerScopingSearchResultsRoot = (props: Props) => {
+  const {meetingRef} = props
+  const meeting = useFragment(
+    graphql`
+      fragment JiraServerScopingSearchResultsRoot_meeting on PokerMeeting {
+        ...JiraServerScopingSearchResults_meeting
+        teamId
+      }
+    `,
+    meetingRef
+  )
+
+  const {teamId} = meeting
+  const queryRef = useQueryLoaderNow<JiraServerScopingSearchResultsQuery>(
+    jiraServerScopingSearchResultsQuery,
+    {
+      teamId,
+      isJQL: false,
+      first: 25
+    }
+  )
+
+  return (
+    <Suspense fallback={''}>
```

Comment:
+1 could add the `MockScopingList` component as the fallback here so that users get some feedback while this query is executing. I'm adding this to the GitHub & GitLab integrations in a PR I'm working on